rnaseq.nf

```diff
@@ -52,12 +29,14 @@ ch_ribo_db = file(params.ribo_database_manifest)
 if (ch_ribo_db.isEmpty()) {exit 1, "File ${ch_ribo_db.getName()} is empty!"}
 
 // Check alignment parameters
-def alignerList       = ['star', 'hisat2', 'star_rsem']
-def pseudoAlignerList = ['salmon']
+def prepareToolIndices  = []
+def alignerList         = ['star_salmon', 'star_rsem', 'hisat2']
+def pseudoAlignerList   = ['salmon']
 if (!params.skip_alignment) {
```

Comment:
I'm not sure if I fully understood this logic... but if the user wanted to use both an aligner and a pseudo-aligner, would the index for the pseudo-aligner be built? (i.e. `--pseudo_aligner=salmon` but NOT `--skip_alignment`)